src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Order completeOrder(ShoppingCart shoppingCart);
```

Comment:
Look at checklist!
Don’t create method completeOrder(ShoppingCart shoppingCart) on the dao layer. Let’s create add(Order order) instead.